server/scripts/create-database.ts

```diff
@@ -32,88 +32,87 @@ function createDatabase(): void {
-    `-U ${envVars.PG_USER} ` +
-    dbName;
+function executeCreateDb(host: string, port: string, user: string, password: string, dbName: string) {
+  const env = Object.assign({}, process.env, { PGPASSWORD: password });
+  const createDbArgs = ['-h', host, '-p', port, '-U', user, dbName];
+  const options = { env, stdio: 'pipe' } as ExecFileSyncOptions;
 
-  exec(createdb, (err, _stdout, _stderr) => {
-    if (!err) {
-      console.log(`Created database ${dbName}\n`);
-      return;
-    }
+  execFileSync('createdb', createDbArgs, options);
+}
 
-    const errorMessage = `database "${dbName}" already exists\n`;
+async function createDb(envVars, dbName): Promise<void> {
+  if (isEmpty(dbName)) {
+    throw new Error('Database name cannot be empty');
+  }
 
-    if (err.message.includes(errorMessage)) {
+  try {
+    executeCreateDb(envVars.PG_HOST, envVars.PG_PORT, envVars.PG_USER, envVars.PG_PASS, dbName);
+    console.log(`Created database ${dbName}\n`);
+  } catch (error) {
+    if (error.message.includes(`database "${dbName}" already exists`)) {
       console.log(`Using Application database\nPG_DB: ${dbName}\nPG_HOST: ${envVars.PG_HOST}\n`);
     } else {
-      console.error(err);
-      process.exit(1);
+      throw error;
     }
-  });
+  }
 }
 
-function createTooljetDb(envVars, dbName) {
-  if (isEmpty(dbName)) throw 'Database name cannot be empty';
-
-  const createdb =
-    `PGPASSWORD="${envVars.TOOLJET_DB_PASS}" createdb ` +
-    `-h ${envVars.TOOLJET_DB_HOST} ` +
-    `-p ${envVars.TOOLJET_DB_PORT} ` +
-    `-U ${envVars.TOOLJET_DB_USER} ` +
-    dbName;
-
-  exec(createdb, (err, _stdout, _stderr) => {
-    if (!err) {
-      console.log(`Created database ${dbName}\n`);
-      return;
-    }
-
-    const errorMessage = `database "${dbName}" already exists\n`;
+async function createTooljetDb(envVars, dbName): Promise<void> {
+  if (isEmpty(dbName)) {
+    throw new Error('Database name cannot be empty');
+  }
 
-    if (err.message.includes(errorMessage)) {
+  try {
+    executeCreateDb(
+      envVars.TOOLJET_DB_HOST,
+      envVars.TOOLJET_DB_PORT,
+      envVars.TOOLJET_DB_USER,
+      envVars.TOOLJET_DB_PASS,
+      dbName
+    );
+  } catch (error) {
+    if (error.message.includes(`database "${dbName}" already exists`)) {
       console.log(`Using Tooljet database\nTOOLJET_DB: ${dbName}\nTOOLJET_DB_HOST: ${envVars.TOOLJET_DB_HOST}\n`);
     } else {
-      console.error(_stderr);
-      process.exit(1);
+      throw error;
     }
-  });
+  }
 }
 
-const nodeEnvPath = path.resolve(process.cwd(), process.env.NODE_ENV === 'test' ? '../.env.test' : '../.env');
-
-const fallbackPath = path.resolve(process.cwd(), '../.env');
-
-if (fs.existsSync(nodeEnvPath)) {
-  createDatabaseFromFile(nodeEnvPath);
-} else if (fs.existsSync(fallbackPath)) {
-  createDatabaseFromFile(fallbackPath);
-} else {
-  console.log('Picking up config from the environment');
-  createDatabase();
+try {
+  checkCommandAvailable('dropdb');
```

Comment:
Command available should be dropdb ? Not sure

Reply:
Nope, it's incorrect. Fixing it.